src/main/java/org/mskcc/cbio/oncokb/config/CacheConfiguration.java

```diff
@@ -29,7 +31,18 @@
 @Configuration
 @EnableCaching
 public class CacheConfiguration {
+
+    private final String PROP_PREFIX = "application.redis";
+    private final String PROP_NAME = "enabled";
+
+    @Bean
+    @ConditionalOnProperty(prefix = PROP_PREFIX, name = PROP_NAME, havingValue = "false")
+    public org.springframework.cache.CacheManager cacheManager(ApplicationProperties applicationProperties){
+        return new CustomCacheManager();
```

Comment:
Since the custom cache manager doesn't really do anything special in the getCache method, we can probably use a cache manager from spring instead (like ConcurrentMapCacheManager).